File: class_primer/class_primer__make.cs
using static System.Console;

internal class Program
{
    class Person
    {
        int Old;
        string Birth;
        string Name;
        string State;

        public Person( string n, int o, string b, string s )
        {
            Name = n;
            Old = o;
            Birth = b;
            State = s;
        }

        public void Print()
        {
            WriteLine( "User{" );
            WriteLine( $"nickname : {Name}" );
            WriteLine( $"old : {Old}" );
            WriteLine( $"birth : {Birth}" );
            WriteLine( $"state : {State}" );
            WriteLine( "}" );
        }
    }

    static void Main()
    {
        int n = int.Parse( ReadLine() );

        for( int _ = 0; _ < n; _++ ) {
            string[] s = ReadLine().Split();
            Person p = new Person( s[0], int.Parse( s[1] ), s[2], s[3] );

            p.Print();
        }
        return;
    }
}

File: class_primer/class_primer__make.py
class user:
    def __init__(self, s):
        self.name = s[0]
        self.old = s[1]
        self.birth = s[2]
        self.state = s[3]

    def write(self):
        print("User{")
        print(f"nickname : {self.name}")
        print(f"old : {self.old}")
        print(f"birth : {self.birth}")
        print(f"state : {self.state}")
        print("}")

n = int(input())

for _ in range(n):
    i = user(input().split())

    i.write()
